InstallFullDB.sh

```diff
@@ -2484,6 +2621,7 @@ function advanced_db_install_menu()
     echo "> 5) Create and fill logs database"
     echo "> 6) Create 'core user' for db and set its default privileges"
     echo "> 7) Delete all databases and users"
+	echo "> 8) Create and fill mangosbots db"
```

Comment:
(nitpick) Irregular indentation (tab where we have spaces otherwise in this bash)